iot/README.md

```diff
@@ -9,4 +9,10 @@ IoT deals with all manner of conventional devices made smart and connected, so t
 
 This requires knowledge of embedded linux (on Raspberry Pi, and other SBCs), Python (The de-facto language for an RPi), microcontrollers, network Connectivity (Firebase, LAN systems), and App Development. The course mentioned is the Android Development Nanodegree, which will give you a fair base to start developing for android. Apart from this, use linux as your daily driver OS, do as much stuff as you can from a terminal (You will mostly SSH into an RPi, no GUI), and you should be fine.Since IoT projects involve wrangling with a whole host of APIs, SDKs and other stuff, there are no courses that encompass it fully, but an app is almost always part of the loop.
 
+## Books
+
+[IoT For Architects](https://drive.google.com/file/d/1QdTIp8jX1EZWd8g0NN01IscJfGnyw2K-/view?usp=sharing)
```

Comment:
Do you really want to add a link to a pirated book on an open platform?

Reply:
we should remove that link and rather add only the book name and amazon link only